lib/offline.js

```diff
@@ -135,8 +135,23 @@ module.exports = function(config) {
     }
 
     function checkImports(imports, rootDir) {
-      imports.forEach(function (filepath) {
-        assertExists(path.join(rootDir, filepath));
+      imports.forEach(function(filepath) {
+        var warning;
+
+        try {
+          if (!fs.statSync(path.join(rootDir, filepath)).isFile()) {
+            warning = 'is a directory';
+          }
+        } catch (ex) {
+          warning = 'doesn\'t exist';
+        }
+
+        if (warning) {
+          process.stdout.write(chalk.yellow.bold(
+            '⚠ ' + filepath + ' ' + warning + ', but you\'ve added it to importScripts.\n' +
+            '  To fix this problem, ensure file/path is a file or remove it from importScripts.\n'
```

Comment:
Erm, when I wrote *file/path* in my suggested warning messages, I meant for that to be a placeholder for the actual file path!

However, in retrospect, it's probably overkill to include the file path twice in this message. So I would simply change "ensure file/path is a file" to "ensure the path is a file."